### ISSUE TITLE
Embedded PDF preview

### ISSUE THREAD

**Issue by linuspahl** [pagedjs/pagedjs#219]
Hi, thank you for this great library.

We are currently trying out paged.js and features like the pagination and table of contents work great when generating a PDF based on a website.

We have one use case where I am not sure if paged.js supports it. We want to create a page which allows you to edit the content of the PDF file while showing a preview of the PDF next to it.

It is always a challenge to display the elements in the preview with the same styles like in the actual PDF file and it looked promising that paged.js is able polyfill the `print` media type.

The problem we face is that paged.js takes all the `print` styles and applies them globally.

For example we have a global style:

```
@media print {
  button {
    background-color: transparent;
  }
}
```

Which is now being applied to all buttons on the page, even those outside of the PDF preview section. I could not find an example for this and I wondering if this has ever been a use case for others?

There is probably more to it, but as far as I can tell, being able to scope the collected styles would solve most of the issue.
Example output:

```
.pdf-preview button {
  background-color: transparent;
}
```

**Comment by julientaq** [comment]
yes, that’s one of the limit of paged.js that we may be able to work on in the future with css layer and scope.

Until then your best option is to set up an iframe that run paged.js. Would that be possible for you?

you can see how we do it (as a very simple example) here: https://gitlab.coko.foundation/pagedjs/parametricpagedjs

**Comment by linuspahl** [comment]
Thank you, I will give it a try. Feel free to close this issue.
